IPython/terminal/interactiveshell.py

```diff
@@ -69,26 +87,24 @@ def get_default_editor():
 
     if os.name == 'posix':
         return 'vi'  # the only one guaranteed to be there!
-    else:
-        return 'notepad' # same in Windows!
+    return 'notepad' # same in Windows!
 
 # conservatively check for tty
 # overridden streams can result in things like:
 # - sys.stdin = None
 # - no isatty method
 for _name in ('stdin', 'stdout', 'stderr'):
     _stream = getattr(sys, _name)
+    _is_tty = True
+    
     if not _stream or not hasattr(_stream, 'isatty') or not _stream.isatty():
         _is_tty = False
         break
-else:
-    _is_tty = True
-
 
 _use_simple_prompt = ('IPY_TEST_SIMPLE_PROMPT' in os.environ) or (not _is_tty)
 
 class TerminalInteractiveShell(InteractiveShell):
-    space_for_menu = Integer(6, help='Number of line at the bottom of the screen '
+    space_for_menu = Integer(6, help ='Number of line at the bottom of the screen '
```

Comment:
I dont' think a space is needed here.

Reply:
You are right, was about remove the space. Forgot about it.